container_files/demos/Convolutions using SQL and Javascript.ipynb

```diff
@@ -0,0 +1,478 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f5b443d9210>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "options=('Right Sobel', 'Detect Edges', 'Sharpen', 'Box Blur', 'Approximated Gaussian Blur')\n",
+    "interact(convolutionFunc, image_processing=kernelDict.keys(), );\n",
+    "print \"Choose an image processing option from the drop-down menu\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I found the 'Detect Edges' convolution particularly useful when training image recognition models. This can be useful in many Machine Vision applications."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Convolutions with MLDB's TensorFlow plug-in\n",
+    "\n",
+    "Not everyone will want to code their own convolutions from scratch (such as with the `create_convolution()` function above). In fact, given the myriad of tools available, it may save you time and effort to use external librairies. MLDB has integrated the TensorFlow Open Source Library for Machine Intelligence allowing us to leverage some of the great Computer Vision APIs that it offers. Let's get started with the same images as before.\n",
+    "\n",
+    "First, I reshape my image and kernel lists into 4D tensors. Then, I use the [`tf_Conv2D`](../../../../doc/builtin/sql/ValueExpression.md.html#builtinfunctions) built-in convolution function."
```

Comment:
use the [`tf_Conv2D`], which is a Tensorflow operator that is exposed as an MLDB built-in function directly in SQL.